extensions/amp-accordion/1.0/amp-accordion.js

```diff
@@ -34,9 +35,29 @@ const HEADER_SHIM_PROP = '__AMP_H_SHIM';
 const CONTENT_SHIM_PROP = '__AMP_C_SHIM';
 const SECTION_POST_RENDER = '__AMP_PR';
 
+/** @extends {PreactBaseElement<AccordionDef.AccordionApi>} */
 class AmpAccordion extends PreactBaseElement {
   /** @override */
   init() {
+    this.registerApiAction(
+      'toggle',
+      (api, invocation) =>
+        api.toggle(invocation.args && invocation.args['section']),
+      ActionTrust.LOW
```

Comment:
Are you sure it's a `LOW` trust here? I'd expect at least a default trust level.

Reply:
Thanks Dima!  Per offline conversation, removing `ActionTrust` to use the default value in `registerAction` which is `ActionTrust.DEFAULT`.

Reply:
That's good. But I think you need to modify `registerApiAction` declaration in the `PreactBaseElement` slightly. Otherwise it will complain about a missing arg. I think you can just add `=` to the `minTrust` type. Run `gulp check-types` to check.

Reply:
Thanks Dima, this is updated.  I also refactored another issue I saw when running `gulp check-types`

`id in expandedMap && !isExpanded(id)` updated to => `!isExpanded(id, true)`
`id in expandedMap && isExpanded(id)` updated to => `isExpanded(id, false)`

this is because isExpanded actually requires two arguments, and already protects for `id in expandedMap`.  Here it is for your reference:
isExpanded = `(id, defaultExpanded) => expandedMap[id] ?? defaultExpanded,`

Summary of the latest changes: https://github.com/ampproject/amphtml/pull/30754/commits/d16497d362547ce302d4ef917e82ed2fb2462ec0